bin/pycbc_multi_inspiral

```diff
@@ -51,6 +51,47 @@ from pycbc.types import zeros, float32, complex64
 from pycbc.vetoes import sgchisq
 
 
+def sky_grid_from_cli(parser, args):
+    """Read the sky grid or the single sky position given the CLI arguments.
+    """
+    if (
+        args.sky_grid is not None
+        and (args.ra is not None or args.dec is not None)
+    ):
+        parser.error('Give either a sky grid or a sky position, not both')
+    if args.sky_grid is not None:
+        with h5py.File(args.sky_grid, 'r') as sky_grid_file:
+            ra = sky_grid_file['ra'][:]
+            dec = sky_grid_file['dec'][:]
+    elif args.ra is not None and args.dec is not None:
+        ra = np.array([args.ra])
+        dec = np.array([args.dec])
+    else:
+        parser.error(
+            'Please specify a sky grid via --sky-grid '
+            'or a single position via --ra and --dec'
+        )
+    return np.array([ra, dec])
+
+
+def calculate_antenna_pattern(args, sky_pos_indices):
+    """Precalculate the antenna pattern functions for all detectors and sky
+    positions.
+    """
+    antenna_pattern = {}
+    for ifo in args.instruments:
+        curr_det = Detector(ifo)
```

Comment:
Tests are complaining about this typo, we should switch  `Detector(ifo)` -> `detector(ifo)`

Reply:
I noticed the `detector.Detector` [class](https://github.com/gwastro/pycbc/blob/0d5f46c6bbbcaeb1dabab442c78640dcde0b6bba/pycbc/detector.py#L194), has this arg "reference_time" that defaults to a GW150914 time. Do we want this ?  otherwise we should use `detector.Detector(ifo, reference_time=None)`

Reply:
I am not sure that will have a big effect, but setting it to None or to the GRB time might be more correct in principle, and might avoid some numerical issues with the antenna pattern or time delay calculations.

I am not going to change this here, but if you want to do a comparison, go ahead.